src/ctia/lib/riemann.clj

```diff
@@ -0,0 +1,115 @@
+(ns ctia.lib.riemann
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [riemann.client :as riemann]
+            [ctia.properties :as prop]
+            [ctia.lib.utils :as utils])
+  (:import [clojure.lang ExceptionInfo]))
+
+(defn request-to-event
```

Comment:
`request->event` is a quite usual naming convention

Reply:
Indeed, 90% of this code is copy-pasted from RiemannReporterService as I get my bearings. Will circle back.